src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/writeText..st

```diff
@@ -0,0 +1,3 @@
+writing
+writeText: anHex
+	self write: anHex charMode: 1
```

Comment:
what is charMode here? What does it control?

Reply:
When charMode is 1, it write text to LCD. When it is 0, it write commands to LCD (clear, change line etc). Because of this behavior, I created 2 methods to use "write: charMode:" function (writeText: and writeCommand:). I never call "write: charMode:" directly. Either the other methods are passing text or commands.